storm/src/main/storm/mesos/NimbusScheduler.java

```diff
@@ -102,7 +102,6 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
       case TASK_LOST:
       case TASK_ERROR:
         LOG.info("Received status update: {}", taskStatusToString(status));
```

Comment:
Perhaps it would make sense to combine `TASK_RUNNING`, `TASK_FINISHED`, and `TASK_KILLED` into an info log line (as those are usually expected), but `TASK_LOST`, `TASK_FAILED`, and `TASK_ERROR` can use a warn log line instead?

Thoughts @erikdw @srishtyagrawal ?